proto/greenfield/storage/query.proto

```diff
@@ -356,4 +370,27 @@ message QueryIsPriceChangedResponse {
     (gogoproto.nullable) = false
   ];
 }
+
+message QueryGroupMembersExistRequest {
+  string group_id = 1;
+  repeated string members = 2 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+}
+
+message QueryGroupMembersExistResponse {
+  repeated bool are_members = 1;
```

Comment:
I think it is better not only boolean, but response the accordingly accounts as well.